src/commands/createVirtualMachine/VirtualMachineCreateStep.ts

```diff
@@ -20,9 +20,11 @@ export class VirtualMachineCreateStep extends AzureWizardExecuteStep<IVirtualMac
     public priority: number = 260;
 
     public async execute(context: IVirtualMachineWizardContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
+        const location = (await LocationListStep.getLocation(context)).name;
```

Comment:
nit: typey type

Reply:
Fwiw, I'm on the side of _not_ needing types everywhere - TypeScript does a lot of type stuff automagically for us without needing to explicitly specify a type. Type enforcement was turned on by default for tslint, but not eslint (aka this is your doing @wwlorey). If we decide we want to enforce types in any repos, tho, we should use eslint to enforce it rather than PR comments IMO

Reply:
I personally like seeing types but not enough to enforce them with a lint rule.. @nturinski up to you on this one, I'm groovy either way

Reply:
```suggestion
        const location: string = (await LocationListStep.getLocation(context)).name;
```